src/css/index.scss

```diff
@@ -0,0 +1,103 @@
+.Info-text1{ 
+    width: 660px;
+    height: 630px;
+    background-image: repeating-linear-gradient(120deg,#3498bd,#8e44ad); 
```

Comment:
why do we need `repeating-linear-gradient`, we don't have this in our example site.